a8c-files.php

```diff
@@ -260,9 +260,9 @@ public function image_resize( $ignore, $id, $size ) {
 			$_max_w = get_option( 'large_size_w' );
 			$_max_h = get_option( 'large_size_h' );
 		} elseif ( is_array( $size ) ) {
-			$_max_w = $size[0];
-			$_max_h = $size[1];
-			$w      = $_max_w;
+			$_max_w = $size[0] ?? 0;
+			$_max_h = $size[1] ?? 0;
+			$w      = $_max_h;
```

Comment:
```suggestion
			$w      = $_max_w;
```

Reply:
Oops, that must have snuck in there! Thanks.